docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,84 @@
 ---
-title: "Administration Console"
+title: "Use the Remote Console"
 date: 2019-02-23T17:39:15-05:00
-draft: true
+draft: false
 weight: 2
+description: "Use the Oracle WebLogic Server Remote Console to manage a domain running in Kubernetes."
 ---
+
+The Oracle WebLogic Server Remote Console is a lightweight, open source console that does not need to be collocated with a WebLogic Server domain.
+You can install and run the Remote Console anywhere. For an introduction, read the blog, "The NEW WebLogic Server Remote Console."
+For detailed documentation, see the [Oracle WebLogic Server Remote Console](https://github.com/oracle/weblogic-remote-console) GitHub project.
+
+A major benefit of using the Remote Console is that you don't need to install or run the WebLogic Server Administration Console on WebLogic Server instances.
+You can use the Remote Console with WebLogic Server **slim** installers, available on the Oracle Technology Network [(OTN)](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html)
+or Oracle Software Delivery Cloud [(OSDC)](https://edelivery.oracle.com/osdc/faces/Home.jspx;jsessionid=LchBX6sgzwv5MwSaamMxrIIk-etWJLb0IyCet9mcnqAYnINXvWzi!-1201085350).
+Slim installers reduce the size of WebLogic Server downloads, installations, Docker images, and Kubernetes pods.
+For example, a WebLogic Server 12.2.1.4 slim installer download is approximately 180 MB.
+
+## Use the Remote Console
+
+To access WebLogic Server domains running in Kubernetes:
+
+1. Install and configure the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+1. For the Remote Console to access the Administration Server running in Kubernetes, you can:
+
+   * Use [curl](#use-curl).
+   * Use the [Administration Server `NodePort`](#use-the-administration-server-nodeport).
+   * Configure [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+### Use curl
+
+Access the REST interface of the WebLogic Server Administration Server to verify the connection and that the correct `hostname:port` is being used:
+
+```shell
+$ curl --user username:password http://host:lbport/console/login/LoginForm.jsp
+```
+
+### Use the Administration Server `NodePort`
+
+To connect to the Remote Console, use the Kubernetes WebLogic Server Administration Server’s `NodePort` :
+
+```shell
+$ http://hostname:adminserver-NodePort/
+```
+
+### Configure ingress path routing rules
+
```

Comment:
These ingress path routing rules allows you to connect to both the On-Server Administration Console as well as the Remote Console.  To connect to the On-Server Administration Console use the following URL in your browserhttp://${HOSTNAME}:${LB_PORT}/console , to connect to the  Remote Console use the following URL in your browser http://${HOSTNAME}:${LB_PORT}/

Reply:
@maggiehe00  can you verify that these are the correct URL constructs to connect?

Reply:
@mriccell I'm not sure what you are suggesting in this comment. In the "Configure ingress path routing rules" section, do you want me to add the URL for connecting to the Remote Console? And, are you suggesting that I add the URL for connecting to the On-Server Administration Console? Do you want this information in addition to what is in the section or as a replacement for it?

Reply:
The doc lacks instructions how to construct the URL for the Remote Console to connect to the Admin Server. 
The ingress path routing rules documented have rules for both the consoles.  We either change the ingress path routing to remove the On-Server Admin Console and only have the Remote console or we need to document both.

Reply:
Monica, I think for the remote console documentation we might want to "change the ingress path routing to remove the On-Server Admin Console  and only have the Remote console". 
Rosemary, I will email you both versions.

Reply:
Monica and I agreed to show the one that has the rules for the Remote Console only.